pom.xml

```diff
@@ -920,6 +926,16 @@
 					<artifactId>schema-to-pojo-maven-plugin</artifactId>
 					<version>${schema-to-pojo.version}</version>
 				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>javacc-maven-plugin</artifactId>
+					<version>2.6</version>
+				</plugin>
+				<plugin>
+					<groupId>org.codehaus.mojo</groupId>
+					<artifactId>build-helper-maven-plugin</artifactId>
+					<version>1.8</version>
```

Comment:
Can you add the same to the schema to pojo? :P

Reply:
It has already been added to schema-to-pojo-integration-tests pom.xml.  Have you run the build from the command line to generate the classes?